### PR TITLE
Refactored Pages:

### PR DESCRIPTION
**Pull request by JamieForster**
- Renamed poll_list.dart to boardroom.dart
- Deleted OLD boardrooms.dart
- Refactored home.dart, moving relevant methods to new boardroom.dart

### CODE REVIEW

File: lib/pages/boardroom.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            ListTile(
+                leading: Icon(Icons.local_hospital, size: 50, color: Colors.white),
+                title: Text(snapshot['pollName'], style: new TextStyle(color: Colors.white)),
+                subtitle: Text(snapshot['patient'], style: new TextStyle(color: Colors.white70)),
+                onTap: (){
+                  Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                        builder: (context) => Poll(user: user, document: snapshot, boardroom: boardroom), // TODO: build navigation to new individual Poll
+                        // currently breaks on tap, use the BACK BUTTON on the emulator to navigate from the error
```
Comment (1solation) [inline lib/pages/boardroom.dart:21]: is this comment still true?

File: lib/pages/boardroom.dart [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                      )
+                  ); // Navigator.push
```
Comment (1solation) [inline lib/pages/boardroom.dart:23]: remove dead comment

**Review by 1solation — CHANGES_REQUESTED** [review]

File: lib/pages/boardroom.dart (hunk 1 above)
Comment (JamieForster) [reply lib/pages/boardroom.dart:21]: 👍 Still true in this build.

File: lib/pages/boardroom.dart (hunk 2 above)
Comment (JamieForster) [reply lib/pages/boardroom.dart:23]: 👍

**Pull request merged**